### PR TITLE
adding filetype for album object

### PR DESCRIPTION
**Pull request by bigbadcapers**
* [x] CDN url updated
* [x] Added entry for `FileIconType.album` and mapping to CDN assets
* [x] Code is up-to-date with the `master` branch
* [x] Your changes are covered by tests: Yes, tested in react-experiments under the FileTypeIcon storybook.
* [x] You've run `yarn change` locally
* [x] style-utilities API updated

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

✅ No changes found
<div id="bundleSizeComment9423"/>

File: change/@fluentui-react-file-type-icons-39aceee7-fe31-4d84-9f6b-bac0108d5d72.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-react-file-type-icons-39aceee7-fe31-4d84-9f6b-bac0108d5d72.json:1]: ## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33167&commitId=6712cc28204a81b62b466f704ca908e36da5dce6&env=prod&clientType=FLUENTUI)


  <details><summary><b>Callout</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.Left center.chromium.png|2616| Changed|
|Callout.Root.chromium.png|2309| Changed|
|Callout.Left top edge.chromium.png|1949| Changed|
|Callout.Gap space 25.chromium.png|2309| Changed|
|Callout.Bottom left edge.chromium.png|2309| Changed|
|Callout.No beak.chromium.png|2306| Changed|
|Callout.Top auto edge.chromium.png|2307| Changed|
|Callout.Top right edge.chromium.png|1594| Changed|

</details>


  <details><summary><b>react-charting-LineChart</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-LineChart.Events.chromium.png|2| Changed|
|react-charting-LineChart.Gaps.chromium.png|1| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

Comment (bigbadcapers) [reply change/@fluentui-react-file-type-icons-39aceee7-fe31-4d84-9f6b-bac0108d5d72.json:1]: some noise false positives appeared in the charting package and the tooltip control, which are unrelated to the album entry for filetype icon

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

File: change/@fluentui-react-file-type-icons-39aceee7-fe31-4d84-9f6b-bac0108d5d72.json (hunk 1 above)
Comment (fabricteam) [inline change/@fluentui-react-file-type-icons-39aceee7-fe31-4d84-9f6b-bac0108d5d72.json:1]: ## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33167&commitId=64765e665bd270ca284822a0a88863d7fcdb9410&env=prod&clientType=FLUENTUI)


  <details><summary><b>Callout</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.Left center.chromium.png|2616| Changed|
|Callout.Left bottom edge.chromium.png|3022| Changed|
|Callout.No callout width specified.chromium.png|2319| Changed|
|Callout.Left top edge.chromium.png|1949| Changed|
|Callout.Right bottom edge.chromium.png|3005| Changed|
|Callout.Top auto edge.chromium.png|2307| Changed|

</details>


  <details><summary><b>Keytip</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Keytip.Offset.chromium.png|121| Changed|

</details>


  <details><summary><b>react-charting-AreaChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-AreaChart.Custom Accessibility.chromium.png|11| Changed|

</details>


  <details><summary><b>react-charting-HeatMapChart</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-HeatMapChart.Basic RTL.chromium.png|372| Changed|
|react-charting-HeatMapChart.Basic Dark Mode.chromium.png|334| Changed|

</details>


  <details><summary><b>react-charting-LineChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-LineChart.Gaps.chromium.png|1| Changed|

</details>


  <details><summary><b>react-charting-VerticalBarChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-VerticalBarChart.Basic - Secondary Y Axis.chromium.png|4| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>

**Review by Jahnp — APPROVED** [review]

**Review by spmonahan — APPROVED** [review]

**Pull request merged**